build-ps/percona-xtradb-cluster.spec

```diff
@@ -723,6 +785,8 @@ install -m 644 "%{malloc_lib_source}" \
 # Remove files we explicitly do not want to package, avoids 'unpackaged
 # files' warning.
 rm -f $RBR%{_mandir}/man1/make_win_bin_dist.1*
+rm -f $RBR%{_bindir}/ps_tokudb_admin
```

Comment:
ps_tokudb_admin is only used for tokudb so I don't think it's needed in PXC since it doesn't support tokudb